client/my-sites/plans-grid/components/header-price.tsx

```diff
@@ -154,6 +161,21 @@ const PlanFeatures2023GridHeaderPrice = ( {
 		( { pricing } ) => pricing.introOffer && ! pricing.introOffer.isOfferComplete
 	);
 
+	if ( isAgencyBlueHostPlan( planSlug ) ) {
+		return (
+			<HeaderPriceContainer discounted={ isAnyVisibleGridPlanDiscounted }>
+				<PlanPrice
+					currencyCode="USD"
+					rawPrice={ 6500 }
+					displayPerMonthNotation={ false }
```

Comment:
Obviously we are missing a layer of data around this. If we don't have an endpoint instance of this plan slug (meaning, nothing sent from `Plans.usePlans`), then we could maybe think about mocking the pricing metadata for this plan in one of the hooks, rather than at UI level this way.

Reply:
Ok. I think all the more reason to not do it here. We can force/mock price & currency from the `usePricingMetaForGridPlans` hook instead, or even higher up from `usePlans`, and make sure we are building the UI just as if working with the real data.